apps/molgenis-components/src/Interfaces/IRefModalData.ts

```diff
@@ -2,5 +2,5 @@ import { ITableMetaData } from "./ITableMetaData";
 
 export interface IRefModalData {
   metadata: ITableMetaData;
-  [primaryKey: string]: string | ITableMetaData;
+  [property: string]: string | ITableMetaData;
```

Comment:
can we please use naming that matches rest from EMX2? What do you mean with 'property'? I did understand 'primaryKey'. Do you mean column? Or graphqlField? Maybe I don't understand the purpose of IRefModalData :-)

Reply:
The problem is that there is not just the primery key, so it can be any amount of properties. I suspect this typing is still incorrect and needs to be finetuned.